outputs.tf

```
@@ -12,3 +12,8 @@ output "aws_kms_key_arn" {
   description = "The Amazon Resource Name (ARN) of the key."
   value       = aws_kms_key.s3.arn
 }
+
+output "aws_kms_key_id" {
+  description = "The Amazon Resource Name (ARN) of the key."
+  value       = aws_kms_key.s3.id
```

```suggestion
  value       = aws_kms_key.s3.key_id
```

I think that its supposed to be `key_id` from these docs: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/kms_key

Does it work in your PR other PR?

---

Yes, it does.                   `"S3_KMS_KEY_ID": "c272cc26-8ad5-4474-8eef-74bcdc19081b"`
 is in the state file when I do 
`  environment {
    variables = {
      COMMIT_BUCKET = module.commit_bucket.id,
      S3_KMS_KEY_ID = module.aws_s3_bucket_kms_key.aws_kms_key_id
    }
  }`
I'm glad I PR'd this, nice eye but now I need to figure out why this works. 🤷

---

It might just be a legacy thing. If `key_id` works then I'd change it so that you are in line with the current tf docs.

---

From the state:
`
      "instances": [
        {
          "schema_version": 0,
          "attributes": {
            "arn": "arn:aws-us-gov:kms:us-gov-west-1:130115508824:key/c272cc26-8ad5-4474-8eef-74bcdc19081b",
....
            "id": "c272cc26-8ad5-4474-8eef-74bcdc19081b",
            "is_enabled": true,
            "key_id": "c272cc26-8ad5-4474-8eef-74bcdc19081b",
            "key_usage": "ENCRYPT_DECRYPT",
          
          },
          "private": "bnVsbA==",
          "dependencies": [
            "module.clonewars.module.aws_s3_bucket_kms_key.data.aws_iam_policy_document.s3"
          ]
`
Going to switch to key_id since it's technically supported while the other is dumb luck.